documentation/src/main/asciidoc/index.asciidoc

```diff
@@ -6,10 +6,10 @@ Hardy Ferentschik; Gunnar Morling; Guillaume Smet
 :xrefstyle: full
 :anchor:
 :toc: left
-:toclevels: 3
+:toclevels: 4
 :sectnumlevels: 5
 :docinfodir: {docinfodir}
-:docinfo:
+:docinfo: shared,private
```

Comment:
This is not really necessary, but apparently it's the new syntax, so I changed this while I was at it.